breba_app/orchestrator.py

```diff
@@ -123,15 +125,21 @@ async def to_generator(user_name: str, session_id: str, message: str, builder_co
 
     if is_task_completed:
         await builder_completed_callback(content)
+        await process_generator_message_callback("__completed__")
```

Comment:
This seems to have the intent of telling the UI that it should now be responsive to user input. Should again be a direct and declarative function call.